client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.scss

```diff
@@ -9,6 +9,15 @@
 	margin-inline-start: 10px;
 	box-sizing: border-box;
 	z-index: 1;
+	transition: margin-inline-start 0.1s ease-out;
+
+	&.device-switcher__container--frame-bordered .device-switcher__frame {
+		box-shadow: none;
+	}
+
+	.pattern-assembler__pattern-panel-list--is-open & {
+		margin-inline-start: 365px;
```

Comment:
I guess we have to increase the margin to balance the whitespace on both sides of the large preview and keep the box-shadow. WDYT?

Reply:
I agree about the margins. I think the shadow doesn't work well.
I did the changes following the design in Figma. Let's wait for Lucas' design review before updating that.

Reply:
Sure 🙂